healthScore/.gitignore

```diff
@@ -0,0 +1,2 @@
+**/__pycache__/*
+.DS_Store
```

Comment:
This file need not be pushed

Reply:
Ok. But without it, cache files will be submitted every time I push to my remote repo. Is that ok?

Reply:
You can keep the .git ignore file locally. But you won't need to push it